state_machine/drivers/pycubedmini/lib/pycubed.py

```diff
@@ -94,23 +95,12 @@ def coildriver_vout(driver_index, projected_voltage):
 """
 Sun Sensor Interface functions
 """
-def lux(sun_sensor_index):
-    """ Return the lux reading for a given sun sensor """
-    if sun_sensor_index == "-Y":
-        return _cubesat.sun_yn.lux
-    elif sun_sensor_index == "-Z":
-        return _cubesat.sun_zn.lux
-    elif sun_sensor_index == "-X":
-        return _cubesat.sun_xn.lux
-    elif sun_sensor_index == "+Y":
-        return _cubesat.sun_yp.lux
-    elif sun_sensor_index == "+Z":
-        return _cubesat.sun_zp.lux
-    elif sun_sensor_index == "+X":
-        return _cubesat.sun_xp.lux
-    else:
-        print(sun_sensor_index, "is not a defined sun sensor.")
-        return
+def sun_vector():
```

Comment:
It looks like you're returning the difference in lux readings between opposite facing sensors in this function -- is there any specific use case for this information? 

Also, do we know for sure there's no use case for a "lux" function?

Reply:
In our attitude determination system the only use for sun sensors is getting an inertial frame sun facing vector. As far as I know that's why we even have them in the first place.

Reply:
Got it! In that case, I'll probably move the lux function to the sun sensor part of the check script, because I still need to be able to individually test sensors. Thanks for clarifying!

Reply:
Sounds good, thanks